otelib/backends/services/dataresource.py

```diff
@@ -18,7 +29,7 @@ def create(self, **kwargs) -> None:
             f"{self.url}{self.settings.prefix}/dataresource",
             json=data.dict(),
             params={"session_id": session_id},
-            timeout=self.settings.timeout,
+            timeout=self.requests_timeout,
```

Comment:
Is this not part of the settings anymore?

Reply:
Why did I ever change things in this strange way?!?! [converting everything back to self.settings.timeout]